src/pages/safety-&-side-effects.astro

```diff
@@ -1,18 +1,270 @@
+                <span class="side-effects-page--commun-list__point desktop--only"></span>
+                Headache
+              </div>
+
+              <div class="side-effects-page--commun-list__item">
+                <img class="side-effects-page--commun-list__item__img" src="/assets/images/safety-and-side-effects/icons/finger-icon.png" alt=" " srcset="">
+                <span class="side-effects-page--commun-list__point desktop--only"></span>
+                Redness around the fingernails
+              </div>
+
+              <div class="side-effects-page--commun-list__item">
+                <img class="side-effects-page--commun-list__item__img" src="/assets/images/safety-and-side-effects/icons/itching-icon.png" alt="" srcset="">
+                <span class="side-effects-page--commun-list__point desktop--only"></span>
+                Itching
+              </div>
+
+            </div>
+
+          </div>
+
+          <h3 class="side-effects-page--section__heading side-effects-page--section__heading__3">These are not all the possible side effects of Koselugo</h3>
+
+          <p class="side-effects-page--section__paragraph">
+            For more information, ask your doctor or pharmacist. Call your doctor for medical advice about side effects. You may report side effects to the FDA at <a class="side-effects__link" href="tel:1-800-FDA-1088">1-800-FDA-1088</a> .
+          </p>
+
+        </div>
+        
+      </Section>
+
+      <section class="side-effects-page--secondary-section">
+        <p class="side-effects-page--disclaimer side-effects-page--disclaimer__overlapping text--center color--aqua">
+          Remember, the information on this page does not replace medical guidance or directions from your doctor. Always talk to them about side effects and ways to manage them.
+        </p>
+  
+        <p class="side-effects-page--paragraph text--center color--aqua">
+          See what people taking Koselugo experienced in a clinical study.
+        </p>
+  
+        <a href="#" class="text--center color--rose">View results now</a>
+        
+      </section>
+      
+      <Section>
+        <Fragment slot="section-title">
+          <span>ARE THERE ANY TESTS NEEDED </span> <span
+            class="color-yellow">BEFORE STARTING KOSELUGO?</span
+          >
+        </Fragment>
+
+        <div class="side-effects-page--section">
+          
+          <p class="side-effects-page--section__paragraph">
+            Make a list of questions you’d like to ask your child’s doctor and information you’d like to share about your child:
+          </p>
+
+          <div class="side-effects-page--steps">
+            <img src="/assets/images/safety-and-side-effects/steps-to-test--mb.png" alt=" ">
+            <img src="/assets/images/safety-and-side-effects/steps-to-test.png" alt=" ">
```

Comment:
Please build this section with html. 
In figma we can export the necessary assets, like this:
https://www.figma.com/file/c2RY4spMOvjNHDLD7VYrth/Koselugo.com?node-id=1584%3A19451&t=ZOElbXGYc7n47ngk-1 

In some viewport size this type of assets appear with a bad quality, Also the text
![image](https://user-images.githubusercontent.com/31933833/228645125-14aac9d6-dd94-42af-bb65-a26cbf9fa4d4.png)